test/modules/backup-filesystem/backup_backend_test.go

```diff
@@ -31,10 +30,26 @@ import (
 	"github.com/weaviate/weaviate/usecases/config"
 )
 
-func Test_FilesystemBackend_Backup(t *testing.T) {
-	t.Run("store backup meta", moduleLevelStoreBackupMeta)
-	t.Run("copy objects", moduleLevelCopyObjects)
-	t.Run("copy files", moduleLevelCopyFiles)
+var override = []string{"", ""}
+
+func Test_FileSystemBackend_Start(t *testing.T) {
+	overrides := [][]string{
+		{"", "", ""},
+		{"", "testBucketPathOverride", " with override"}, // Note:  no bucket parameter, because it is not supported by the filesystem backend
+	}
+
+	override = overrides[0]
+	FilesystemBackend_Backup(t)
+	time.Sleep(5 * time.Second)
+
+	override = overrides[1]
+	FilesystemBackend_Backup(t)
+}
+
+func FilesystemBackend_Backup(t *testing.T) {
```

Comment:
can we introduce descriptive params to this method? using global `override` array is quite difficult to decipher the meaning

Reply:
I'm glad you agree, let's apply this approach in other parts of the code.